06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,142 @@
+    const searchForAliceInAllFiles = invertedIndex
+    .searchFiles(['alice'], 'All');
+    const searchForPartyInAllFiles = invertedIndex
+    .searchFiles(['party'], 'All');
+    const searchForAliceInBook = invertedIndex
+    .searchFiles(['alice'], 'Book');
+    const searchForPartyInNews = invertedIndex
+    .searchFiles(['party'], 'News');
+
+    it('should return Object as type of search result', () => {
+      expect(searchForAliceInAllFiles instanceof Object).toBeTruthy();
+    });
+
+    it('should match searchForAliceInAllFiles with alice: { 0: true }', 
+    () => {
+      expect(searchForAliceInAllFiles.Book).toEqual(jasmine.objectContaining({
+        alice: { 0: true }
+      }));
+    });
+
+    it('should match searchForAliceInAllFiles with alice: { 0: false }', () => {
+        expect(searchForAliceInAllFiles.Book).not.toEqual(jasmine.objectContaining({
+          alice: { 0: false }
+        }));
+    });
+    
+    it('should matchsearchForPartyInAllFiles with party: { 0: true, 5: true }', () => {
+      expect(searchForPartyInAllFiles.News).toEqual(jasmine.objectContaining({
+        party: { 0: true, 5: true }
+      }));
+    });
+
+    it('should match searchForPartyInAllFiles with party: { 0: true, 1: true }', 
+      () => {
+        expect(searchForPartyInAllFiles.book).not.toEqual(jasmine.objectContaining({
+          party: { 0: true, 1: true }
+      }));
+    });
+
+    it('should return Object as type of searchResult3', 
+      () => {
+      expect(searchForAliceInBook instanceof Object).toBeTruthy();
+    });
+
+    it('should match searchForAliceInBook with alice: { 0: true }', () => {
+      expect(searchForAliceInBook.Book).toEqual(jasmine.objectContaining({
+        alice: { 0: true }
+      }));
+    });
+
+    it('should match searchForAliceInBook with alice: { 0: false }', 
+      () => {
+      expect(searchForAliceInBook.Book).not.toEqual(jasmine.objectContaining({
+        alice: { 0: false }
+      }));
+    });
+
+    it('should return Object asype of searchResult4', 
+      () => {
+      expect(searchForPartyInNews instanceof Object).toBeTruthy();
+    });
+
+    it('should match searchForPartyInNews with  party: { 0: true, 5: true }', 
+      () => {
+      expect(searchForPartyInNews.News).toEqual(jasmine.objectContaining({
+        party: { 0: true, 5: true }
+      }));
+    });
+    });
+
+    describe('the inverted index getIndex method', () => {
+      const getCorrectBookIndex = invertedIndex.getIndex('Book')
+      const getEmptyIndex = invertedIndex.getIndex()
+      it('should return object as typeof getCorrectBookIndex', () => {
+        expect(getCorrectBookIndex instanceof Object).toBeTruthy();
+      }); 
+
+      it('should return false for getEmptyIndex', () => {
+        expect(getEmptyIndex).toBeFalsy();
+      }); 
+    });
+
+    describe('the inverted index validate method', () => {
+
+      const emptyFileContent = invertedIndex.isValid(empty);
```

Comment:
Block must not be padded by blank lines. - Eslint violation on your describe function.